### PR TITLE
Feat: Adds remainderNumbers method

### PR DESCRIPTION
**Pull request by RhyenSftwareEngr**
**Description**
Adds a new feature remainderNumbers in Main.java

**Changes**
Added remainderNumbers method in Main.java

**Notes For Reviewers:**
Please Review The implementation of remainderNumbers method

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -25,4 +25,7 @@ public int divideNumbers(int a, int b) {
     System.out.println("Dividing numbers: " + a + " / " + b);
     return a/b;
 }
+public int remainderNumber(int a, int b){
```
Comment (AljayAgustin) [inline Main.java:28]: I suggest that you change the naming convention from remainderNumber to remainder_Number.

Comment (AljayAgustin) [inline Main.java:28]: I suggest that you change the naming convention from remainderNumber to remainder_Number.

Comment (RhyenSftwareEngr) [reply Main.java:28]: Noted

File: Main.java
Comment (AljayAgustin) [inline Main.java:1]: that is okay to me

**Pull request merged**